hh-plus-plus.js

```diff
@@ -3086,6 +3088,17 @@ function moduleLeague() {
     const oldScore = JSON.parse(localStorage.getItem('leagueScore')) || {points: 0, avg: 0}
     const {points: oldPoints} = oldScore
     if (playerCurrentPoints > oldPoints) {
+        const player=playerLeaguesData.id_member;
+        const points=playerCurrentPoints-oldPoints;
```

Comment:
Worried about this as a means of determining the individual score. If people use multiple devices to do fights (e.g. desktop and mobile), the oldScore on one won't have updated for fights done on the other, so the first fight you do back on the first device will attribute all those points to a single fight.

Reply:
Additionally, this code being here relies on the correct player being highlighted by default when loading up the league table, which in most cases would be fine, but if the user navigates away via the home button or links at the top and then comes back to the table after the next snapshot, the points could well be attributed to the wrong player

Reply:
I don't think much can be done for multiple devices since it relies on local storage, but I can add a protection to not include any score above a 25.